Backend/dev/migrations/rider-app/1252-add-merchant-id-to-whitelistorg.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE atlas_app.white_list_org ADD COLUMN merchant_id character varying(36) NOT NULL DEFAULT ''::character varying;
+
+-- whitelist merchant level
+WITH MerchantIds AS (
+    SELECT id as merchant_id FROM atlas_app.merchant
+)
+INSERT INTO atlas_app.white_list_org (id, subscriber_id, domain, merchant_id)
+SELECT
+    md5(random()::TEXT || clock_timestamp()::TEXT || random()::TEXT || random()::TEXT || random()::TEXT || random()::TEXT)::UUID,
+    subscriber_id,
+    domain,
+    MerchantIds.merchant_id as merchant_id
+FROM atlas_app.white_list_org CROSS JOIN MerchantIds;
```

Comment:
Add comment which to run before and after migration

Reply:
Fixed.